modules/gitops-release-notes-1-8-5.adoc

```diff
@@ -0,0 +1,34 @@
+/ Module included in the following assembly:
+//
+// * gitops/gitops-release-notes.adoc
+:_content-type: REFERENCE
+[id="gitops-release-notes-1-8-5_{context}"]
+= Release notes for {gitops-title} 1.8.5
+
+{gitops-title} 1.8.5 is now available on {product-title} 4.10, 4.11, 4.12, and 4.13.
+
+[id="errata-updates-1-8-5_{context}"]
+== Errata updates
+
+=== RHSA-2023:120046 - {gitops-title} 1.8.5 security update advisory
+
+Issued: 2023-09-07
+
+The list of security fixes that are included in this release is documented in the following advisory:
+
+* link:https://errata.engineering.redhat.com/advisory/120046[RHBA-2023:120046]
```

Comment:
```suggestion
* link:https://access.redhat.com/errata/120046[RHBA-2023:120046]
```
@masauso-lungu  Please remember to notice the change in the RHSA number if any for this advisory once the errata state is changed to SHIPPED LIVE.